Upstream-Status: Pending

This patch fixes following kind of issues with automake 1.12
| automake: warnings are treated as errors
| data/ui/Makefile.am:21: warning: '%'-style pattern rules are a GNU make extension
...
| /srv/home/nitin/builds/build-gcc47/tmp/sysroots/x86_64-linux/usr/share/automake-1.12/am/ltlibrary.am: warning: 'libfm-gtk.la': linking libtool libraries using a non-POSIX
| /srv/home/nitin/builds/build-gcc47/tmp/sysroots/x86_64-linux/usr/share/automake-1.12/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'
...
| src/Makefile.am:21: warning: compiling 'glib-compat.c' with per-target flags requires 'AM_PROG_CC_C_O' in 'configure.ac'

Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>
2012/05/03

Index: libfm-0.1.17/configure.ac
===================================================================
--- libfm-0.1.17.orig/configure.ac
+++ libfm-0.1.17/configure.ac
@@ -4,11 +4,17 @@
 AC_PREREQ([2.63])
 AC_INIT([libfm], [0.1.17], [http://pcmanfm.sorceforge.net/])
 AM_INIT_AUTOMAKE([-Wall -Werror foreign])
+
+# automake 1.12 seems to require this, but automake 1.11 doesn't recognize it
+m4_pattern_allow([AM_PROG_AR])
+AM_PROG_AR
+
 AC_CONFIG_MACRO_DIR(m4)
 AC_CONFIG_HEADERS([config.h])
 
 # Checks for programs.
 AC_PROG_CC
+AM_PROG_CC_C_O
 AM_PROG_LIBTOOL
 
 # Checks for libraries.
Index: libfm-0.1.17/data/ui/Makefile.am
===================================================================
--- libfm-0.1.17.orig/data/ui/Makefile.am
+++ libfm-0.1.17/data/ui/Makefile.am
@@ -18,6 +18,6 @@ EXTRA_DIST = \
 	$(NULL)
 
 # Purge GtkBuilder UI files
-%.ui: %.glade
+.glade.ui:
 	cp $< $@
 	$(top_builddir)/src/xml-purge $@
